.github/ISSUE_TEMPLATE/variable-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Variable request
+about: Suggest a variable for cchsflow
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is the variable an existing CCHS variable, or a derived variable?**
+Does this variable exist already in CCHS cycles, or is it a newly derived variable?
+
+**What is the name of the variable?**
+What is the most consistent name for this variable (usually the variable name used from 2007-2014). If this is an derived variable, what is the name of the newly derived variable?
+
+**Description of variable**
+Provide a brief description of the variable.
+
+**Is it consistent across CCHS cycles? If not, explain changes between cycles**
+For existing variables in the CCHS.
+
+**Which cycles is this variable found?**
+For existing variables in the CCHS.
+
+**Derived variables only. What variables are used to create this variable?**
+List the variables in cchsflow used to create this variable
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+**Additional instructions for derived variables**
+Please attach an .R file that includes documentation of the derived variable as per roxygen2 standards, along with the code to derive the variable
```

Comment:
"Derived variables use R code for more complex operations and multiple starting variables. Note: cchsflow currently does not include dependancy for derived variables. 
If possible, attach an .R file that includes documentation of the derived variable as per roxygen2 standards, along with the code to derive the variable. Include all starting variables."